bang/magicollections.py

```diff
@@ -1944,14 +1943,15 @@ def to_fields(self, item, *args, **kwargs):
                 diff_value = ''
                 diff = getattr(item, u'{}_difficulty'.format(difficulty), None)
                 if diff is not None:
-                    diff_value = generateDifficulty(diff, True)
-                if getattr(item, u'{}_notes'.format(difficulty), None) != None: diff_value+=_(u'{} notes').format(getattr(item, u'{}_notes'.format(difficulty), None))
+                    diff_value = mark_safe(u'{}<br />'.format(generateDifficulty(diff)))
```

Comment:
You're calling `mark_safe` later so you shouldn't call it here.

Reply:
✅